### PR TITLE
Kallistobus

### PR DESCRIPTION
**Pull request by astrovsky01**
Add in datatypes for kallisto bus and bustools

### CODE REVIEW

File: config/datatypes_conf.xml.sample [hunk 1]
```diff
@@ -367,6 +367,8 @@
       <display file="igv/vcf.xml"/>
       <converter file="vcf_bgzip_to_tabix_converter.xml" target_datatype="tabix"/>
     </datatype>
+    <datatype extension="ec" type="galaxy.datatypes.binary:Binary" subclass="true" display_in_upload="true" />
```
Comment (mvdbeek) [inline config/datatypes_conf.xml.sample]: This and idx are very short extensions -- is there maybe an alternative to it ?
`idx` also seems very general

**Review by mvdbeek — COMMENTED** [review]
Could you maybe drop the merge commits from this branch ?

File: config/datatypes_conf.xml.sample (hunk 1 above)
Comment (astrovsky01) [reply config/datatypes_conf.xml.sample]: idx is the output filetype of part of kallisto tools and input for another. I could add hidden_in_upload?

**Comment by astrovsky01** [comment]
@mvdbeek how do I do that?

File: config/datatypes_conf.xml.sample (hunk 1 above)
Comment (mvdbeek) [reply config/datatypes_conf.xml.sample]: That's not my concern, the extensions need to be unique across Galaxy and users should be able to tell what file it is based on the extension. How about kallisto.ec and kallisto.idx ?

Comment (astrovsky01) [reply config/datatypes_conf.xml.sample]: Ah. I can do that.

**Review by mvdbeek — APPROVED** [review]
Thanks a lot, looks good to me.

**Pull request merged**